lib/suspenders/version.rb

```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "4.2.1"
+  RAILS_VERSION = "4.2.3"
```

Comment:
@taboularasa Agreed. I vote to split this out as separate issue for enhancement?